nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-policy-management.js

```diff
@@ -1611,60 +1611,6 @@
             return loadPolicy().always(showPolicy);
         },
 
-        /**
-         * Shows the template policy.
-         *
-         * @param d
-         */
-        showTemplatePolicy: function (d) {
-            initializingComponentPolicy = true;
-
-            // reset the policy message
-            resetPolicyMessage();
-
-            // update the policy controls visibility
-            $('#component-policy-controls').show();
-            $('#global-policy-controls').hide();
-
-            // update the visibility
-            if (d.permissions.canRead === true) {
-                $('#policy-selected-template-container div.policy-selected-component-name').text(d.template.name);
-            } else {
-                $('#policy-selected-template-container div.policy-selected-component-name').text(d.id);
-            }
-            $('#policy-selected-template-container').show();
-
-            // populate the initial resource
-            $('#selected-policy-component-id').text(d.id);
-            $('#selected-policy-component-type').text('templates');
-            $('#component-policy-target')
-                .combo('setOptionEnabled', {
-                    value: 'operate-component'
-                }, false)
-                .combo('setOptionEnabled', {
-                    value: 'write-receive-data'
-                }, false)
-                .combo('setOptionEnabled', {
-                    value: 'write-send-data'
-                }, false)
-                .combo('setOptionEnabled', {
-                    value: 'read-data'
-                }, false)
-                .combo('setOptionEnabled', {
-                    value: 'read-provenance'
-                }, false)
-                .combo('setOptionEnabled', {
-                    value: 'write-data'
-                }, false)
-                .combo('setSelectedOption', {
-                    value: 'read-component'
-                });
-
-            initializingComponentPolicy = false;
-
-            return loadPolicy().always(showPolicy);
-        },
-
```

Comment:
I don't think we should remove this. This isn't the same kind of template that we are trying to remove.

Reply:
Not so sure now, @scottyaslan do you know?

Reply:
Yeah, apologies if I was heavy handed here. 'Template' the nifi term and html/js 'templates' were causing me some confusion. I'll investigate some more but interested to see what @scottyaslan says.

Reply:
Yeah, i just can't remember if this is a policy for a NiFi Template or a template for a NiFi Policy. If it is the former, we would want to remove it. If it is the later, we would want to keep it.

Reply:
I am building main now to see if I can spark some recollection.

Reply:
OK, I think I was wrong initially. Based on the rest of the surrounding code, i believe this to be showing a policy dialog for an existing template.
There are analogous methods named `showControllerServicePolicy`, `showParameterContextPolicy`, ..., and `showComponentPolicy`. It would then make sense that `showTemplatePolicy` would be used to show the policy of a specific template.

So, I think it is good that we are deleting this.